src/main/java/com/trueffect/logica/Generator.java

```diff
@@ -0,0 +1,55 @@
+package com.trueffect.logica;
+
+import com.trueffect.model.Person;
+
+/**
+ *
+ * @author santiago.mamani
+ */
+public class Generator {
+
+    public static String getStringSet(Person person) {
```

Comment:
What is the objective of this class

Reply:
To generate sql either to modify or for conditions. Example to modify would generate a SET in the following way... "type_ indedifier=CI, identifier=12344456, lastName='Zeballos Arnez'"